daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -221,6 +254,21 @@ func (l *logStream) create() error {
 	return err
 }
 
+// create creates log group and log stream for the instance of the awslogs logging driver
+func (l *logStream) create() error {
```

Comment:
nit: Can you move `create()` above `createLogGroup()` just though things are ordered roughly caller above callee?